terraform/cloudfoundry/cf_rds.tf

```diff
@@ -40,7 +40,7 @@ resource "aws_db_instance" "cf" {
   allocated_storage    = 10
   engine               = "postgres"
   engine_version       = "9.5.4"
-  instance_class       = "db.t2.small"
+  instance_class       = "db.m4.large"
```

Comment:
The cost is significant and `t2.medium` is sufficient.
As a reference, this developer has a similar configuration with all 3 DBs in the same RDS instance and they use `t2.medium`: https://github.com/cloudfoundry/bbs/issues/13#issuecomment-263539941